Modules/MinimapIcon.lua

```diff
@@ -57,6 +57,8 @@ function _MinimapIcon:CreateDataBrokerObject()
                 end
 
                 return;
+            elseif button == "LeftButton" and QuestieMenu:isOpen() then
+                QuestieMenu:Hide()
```

Comment:
I think I'd prefer if this will be added to the previous case.

So that it is somewhat like:

```lua
if button == "LeftButton" then
    if QuestieMenu.IsOpen() then
        QuestieMenu:Hide()
        return
    end
    ...
elseif button == "RightButton" then
    ...
end
```